internal/exec/vendor_model.go

```
@@ -0,0 +1,328 @@
+		packages:  vendorPks,
+		spinner:   s,
+		progress:  p,
+		dryRun:    dryRun,
+		cliConfig: cliConfig,
+		isTTY:     tty,
+	}, nil
+}
+
+func (m modelVendor) Init() tea.Cmd {
+	if len(m.packages) == 0 {
+		m.done = true
```

golangci-lint

SA4005: ineffective assignment to field modelVendor.done

(staticcheck)